opengrok-indexer/src/main/java/org/opengrok/indexer/history/PerforceRepository.java

```diff
@@ -61,14 +62,44 @@ public PerforceRepository() {
         ignoredFiles.add(".p4config");
     }
 
+    public static String protectPerforceFilename(String name) {
```

Comment:
this method could be protected perhaps or package private

Reply:
As it's called from PerforceHistoryParser, I assumed it needed to be public.  Is there a relationship I'm missing?

Reply:
The method can be package private (i.e. omit all the specifiers so the method signature becomes just `static String protectPerforceFilename(String name)`) because it is used by the class itself and also by the test case which resides in the same package.

If you use any decent IDE for editing the sources, it will suggest the change in fact.

Reply:
Well, I'm old school, my IDE is vi.  :-)  Thanks, change made.